lobster/tools/codebeamer/codebeamer.py

```diff
@@ -210,17 +253,50 @@ def import_tagged(mh, cb_config, items_to_import):
     return rv
 
 
+def ensure_array_of_strings(instance):
+    if isinstance(instance, list) and all(isinstance(item, str) for item in instance):
+        return instance
+    else:
+        return [str(instance)]
+
+
+def parse_cb_config_file(file_name):
+    assert isinstance(file_name, str)
+    assert os.path.isfile(file_name)
+
+    with open(file_name, "r") as file:
+        data = json.loads(file.read())
+
+        provided_config_keys = set(data.keys())
```

Comment:
You can unindent everything after `json.loads(file.read())`. The file can be closed immediately once reading the file has finished.